src/main/front/src/components/GoogleLoginButton.test.tsx

```
@@ -0,0 +1,64 @@
+// window.location.href 모킹용
+const originalLocation = window.location;
+
+describe("GoogleLoginButton", () => {
+  beforeEach(() => {
+    // fetch와 alert를 초기화
+    global.fetch = vi.fn();
+    global.alert = vi.fn();
+
+    // window.location.href mock
+    delete window.location;
+    window.location = { href: "" } as Location;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)